### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: On deleting the "Artifact" in push image dialog, the entire dialog becomes not accessible. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1508]
**Repro Steps:**

1. Create a new dynamic web project.
2. Tab to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. Tab to and select the remove button, to delete the artifact,  then observe the keyboard behaviour.

**Expected Result:**
After deleting the Artifact the keyboard focus should stay on delete button and should be able to tab with tab key.

**Actual Result:**
No keyboard action is taking after deleting  the "Artifact"  in push image dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![no keyboard action after deleting artifact in push image](https://user-images.githubusercontent.com/22956130/36146214-c1d553f8-1068-11e8-8c93-5033f2f30961.png)

**Comment by Eskibear** [comment]
Content hierarchy is provided by IntelliJ run/debug configuration, out of the toolkit''s control.

**Comment by Eskibear** [comment]
Reference: https://youtrack.jetbrains.com/issue/IDEA-191322

Tracked in #1627

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. The "Before launch" tasks are auto-generated by IntelliJ IDEA, and users don't need to change/delete it.
* **Quality of experience**: not impacted. The "Delete" button cannot be selected using tab key, but the task item itself can be selected. Pressing "Delete" key on the item is an available workaround.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749987
